### DISCUSSION TITLE
fr: for file picker, support showing modified time

### DISCUSSION THREAD

**Discussion by LaunchGemini** [charmbracelet/huh#554] [Ideas]
Like `ShowSize()`

**Comment by bashbunni** [comment]
Hey @LaunchGemini, thank you so much for wanting to make this project better! I've converted this to a discussion so we can talk through it and keep our backlog clear. If it does end up being something we want to add, we'll promote it back to an issue when we are ready to work on it.

If we commit to this feature, that's a feature we will need to maintain for the lifespan of this project. So, since "Yes is forever", we are carefully considering new features.
